docs/src/pages/docs/plugins/css.md

```diff
@@ -319,4 +319,4 @@ To use this, replace this plugin with @cobalt-ui/plugin-sass in `tokens.config.m
   };
 ```
 
-This changes `token('color.blue')` to return CSS vars rather than the original values. To learn more, [read the dos](../plugin-sass).
```

Comment:
🤔 I could probably fix this more intelligently in the script that copies READMEs > docs. But I’m lazy.